lib/elixir/test/elixir/code_test.exs

```diff
@@ -293,6 +293,14 @@ defmodule CodeTest do
       Code.compiler_options(debug_info: :not_a_boolean)
     end
   end
+
+  describe "fecth docs handles special cases correct" do
```

Comment:
There is a [`docs_test.exs` file](https://github.com/elixir-lang/elixir/blob/master/lib/elixir/test/elixir/kernel/docs_test.exs), I would put this test there :)

Reply:
Makes sense, don't know why I didn't see that file before.